docs/02_types/scalars.md

```diff
@@ -23,31 +25,21 @@ Scalar types represent concrete values at the leaves of a query. For example in
 }
 ```
 
-Strawberry will automatically map the Python built in types to the corresponding
-GraphQL type:
+There are several built-in scalars, and you can define custom scalars too.
+([Enums](/docs/types/enums) are also leaf values.) The built in scalars are:
```

Comment:
`leaf values` would it be useful to explain what a leaf value is?

Reply:
Yeah it probably would. How would you describe it though?

Reply:
No idea, maybe we can leave like it is for now :)